notebooks/202-vision-superresolution/202-vision-superresolution-image.ipynb

```diff
@@ -807,24 +765,22 @@
     "    f\"{num_patches/duration:.2f}.\\nInference patches per second: \"\n",
     "    f\"{num_patches/total_inference_duration:.2f} \"\n",
     ")"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {
+    "tags": []
+   }
   },
   {
    "cell_type": "markdown",
-   "id": "b44ece78",
-   "metadata": {},
    "source": [
     "### Save superresolution image and the bicubic image"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "304b23f8",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
    "source": [
     "full_superresolution_image_path = IMAGE_PATH.with_name(\n",
```

Comment:
The output files for this notebook should be saved in the output directory now. This is one, could you check the notebook for all outputs?